airflow/providers/google/cloud/transfers/bigquery_to_mssql.py

```diff
@@ -39,86 +37,39 @@ class BigQueryToMsSqlOperator(BaseOperator):
         For more information on how to use this operator, take a look at the guide:
         :ref:`howto/operator:BigQueryToMsSqlOperator`
 
-    .. note::
-        If you pass fields to ``selected_fields`` which are in different order than the
-        order of columns already in
-        BQ table, the data will still be in the order of BQ table.
-        For example if the BQ table has 3 columns as
-        ``[A,B,C]`` and you pass 'B,A' in the ``selected_fields``
-        the data would still be of the form ``'A,B'`` and passed through this form
-        to MSSQL
-
-    **Example**: ::
-
-       transfer_data = BigQueryToMsSqlOperator(
-            task_id='task_id',
-            source_project_dataset_table='my-project.mydataset.mytable',
-            mssql_table='dest_table_name',
-            replace=True,
-        )
-
     :param source_project_dataset_table: A dotted ``<project>.<dataset>.<table>``:
         the big query table of origin
-    :param selected_fields: List of fields to return (comma-separated). If
-        unspecified, all fields are returned.
-    :param gcp_conn_id: reference to a specific Google Cloud hook.
+    :param mssql_table: target MsSQL table (templated)
```

Comment:
I think it's best to deprecate this parameter and replace it with a unified parameter in the base class (`target_table_name`)

Reply:
Sure. I changed that.